sorting/bubbleSort.js

```diff
@@ -0,0 +1,18 @@
+const bubbleSort = (arr) => {
+
+    if(arr && Array.isArray(arr)) {
```

Comment:
nope. I meant if a function does not do anything if arguments don't satisfy pre-requisites then return it in second line. In your code, you do not do anything if the input is not an array but that suspense keeps the reader read till line 14.
Suggested way
```javascript
if(!arr || !Array.isArray(arr)) {
return arr;
}
```
in this way you have finished that suspense in second line.